src/libraries/System.Private.Xml/src/System/Xml/Dom/DocumentXPathNavigator.cs

```diff
@@ -180,10 +180,6 @@ public override string Value
                         return ValueText;
                     default:
                         Debug.Assert(_source.Value != null);
-                        // TODO-NULLABLE: Consider switching this.Value to nullable even if that implies switching it in the base type.
```

Comment:
this didn't seem possible to me in legitimate scenario and I found this not worth investigating without actual repro